web/router.ex

```diff
@@ -11,9 +11,7 @@ defmodule RemoteRetro.Router do
     plug :fetch_flash
     plug :protect_from_forgery
     plug :put_secure_browser_headers
-    if Application.get_env(:remote_retro, :add_mock_user_to_session) do
-      plug :add_mock_user_to_session
-    end
+  
```

Comment:
i believe so, yeah. to actually authenticate, Google needs to be able to POST to a specific endpoint with the appropriate `code` param, and they can't make POST requests to your `localhost` unless you expose a port via ngrok. So this `add_mock_user_to_session` plug is only used in development.